src/pages/Admin/Room/Edit/RoomEditModal.tsx

```diff
@@ -67,8 +67,13 @@ export const RoomEditModal: React.FC<RoomEditModalProps> = ({
   );
 
   const onSubmit = useCallback(() => {
-    return submitHandler(values, room.roomIndex!);
-  }, [room.roomIndex, submitHandler, values]);
+    if (room.roomIndex === undefined) return;
```

Comment:
We tend to keep a space after these sorts of 'guard statements' to make them stand out more:
```suggestion
    if (room.roomIndex === undefined) return;

```